### PR TITLE
Small change to include metatranscriptomic

### PR DESCRIPTION
**Pull request by charles-cowart**

### CODE REVIEW

File: sequence_processing_pipeline/QCJob.py [hunk 1]
```diff
@@ -208,9 +208,11 @@ def _process_sample_sheet(self):
...
             raise PipelineError(s)
 
-        needs_adapter_trimming = (True if
-                                  header['Assay'] == 'Metagenomic'
-                                  else False)
+        meta_types = [Pipeline.METAGENOMIC_PTYPE,
+                      Pipeline.METATRANSCRIPTOMIC_PTYPE]
+
+        needs_adapter_trimming = (
+            True if header['Assay'] in meta_types else False)
```
Comment (antgonza) [inline sequence_processing_pipeline/QCJob.py:215]: I don't think `meta_types` is an accurate name; perhaps easier to just do:

`needs_adapter_trimming = header['Assay'] in [Pipeline.METAGENOMIC_PTYPE, Pipeline.METATRANSCRIPTOMIC_PTYPE]`

**Review by antgonza — CHANGES_REQUESTED** [review]
Any tests needed for this change?

**Comment by charles-cowart** [comment]
> Any tests needed for this change?

It might be a little overkill but I added one that confirms Metatranscriptomic is a valid Assay type while an obviously invalid one raises an Error.

File: sequence_processing_pipeline/QCJob.py [hunk 2]
```diff
@@ -208,11 +208,10 @@ def _process_sample_sheet(self):
             s = "Assay value '%s' is not recognized." % header['Assay']
             raise PipelineError(s)
 
-        meta_types = [Pipeline.METAGENOMIC_PTYPE,
-                      Pipeline.METATRANSCRIPTOMIC_PTYPE]
-
         needs_adapter_trimming = (
-            True if header['Assay'] in meta_types else False)
+            True if header['Assay'] in [Pipeline.METAGENOMIC_PTYPE,
```
Comment (antgonza) [inline sequence_processing_pipeline/QCJob.py:212]: I don't think the if/else is necessary here; could you remove it?

**Review by antgonza — APPROVED** [review]

**Pull request merged**